src/main/java/it/gov/pagopa/payhub/auth/utils/SecurityUtils.java

```diff
@@ -24,8 +24,21 @@ public static List<String> getPrincipalRoles(String organizationIpaCode){
                 .orElse(Collections.emptyList());
     }
 
+    /** It will return user's session roles on requested organization IPA code retrieving it from ThreadLocal */
```

Comment:
this javadoc is not aligned with the logic.
Moreover, I cannot see how just this method could be util...
I cannot see other use other than to check if there is at least 1 ROLE_ADMIN

You could optimize this stream operator using anyMatch operator instead of cycling the entire list